tests/core/test_feedback.py

```diff
@@ -0,0 +1,33 @@
+"""Test UtteranceFeedback class."""
+import pytest
+
+from dialoguekit.core.feedback import BinaryFeedback, UtteranceFeedback
+
+
+def test_initialization():
+    """Tests utterance feedback initialization."""
+    f1 = UtteranceFeedback("utterance_1", BinaryFeedback.POSITIVE)
+    assert isinstance(f1, UtteranceFeedback)
+    assert f1.feedback == BinaryFeedback.POSITIVE
+
+
+def test_hash():
```

Comment:
Not sure this test is needed, `unsafe_hash=True` implies that `__hash__()` is generated for `UtteranceFeedback`.

Reply:
Removed